--- a/dspace-spring-rest/src/main/java/org/dspace/app/rest/model/RootRest.java
+++ b/dspace-spring-rest/src/main/java/org/dspace/app/rest/model/RootRest.java
@@ -25,6 +25,10 @@ public class RootRest implements RestAddressableModel {
         return CATEGORY;
     }

+    public String getComment() {
+        return "Hello OR2019!";
+    }
+
     public String getType() {
         return NAME;
     }
